specs/signature-envelope-jws.md

```diff
@@ -54,6 +64,21 @@ Example of the Notary Project signature payload
 }
 ```
 
+Example of the Notary Project blob signature payload:
+
+```jsonc
+{
+  "targetArtifact": {
+    "mediaType": "application/octet-stream",
+    "digest": "sha256:2f3a23b6373afb134ddcd864be8e037e34a662d090d33ee849471ff73c873345",
+    "size": 1024,
+    "annotations": {
+        "io.wabbit-networks.buildId": "123"  // user defined metadata
+    }
+  }
+}
+```
+
```

Comment:
Payload format is same for blob and OCI. IMO we dont need desperate example for blob and oci